.gitignore

```diff
@@ -6,6 +6,7 @@
 /lib/elixir/src/*_parser.erl
 /lib/elixir/src/elixir.app.src
 /lib/elixir/test/ebin
+/lib/elixir/_build
```

Comment:
Ahh, ok.  I think  tried to just run the elixir tests using "mix"; I'll remove this and look a little harder at the testing docs for how to run tests just against one lib when debugging.